### ISSUE TITLE
PropertyGrid Toolstrip - ToolStripButtons Selected Border Mouse Over

### ISSUE THREAD

**Issue by AngeloCresta** [dotnet/winforms#13210]
**.NET Core Version:**
9.0.201

**Did it work in .NET Framework?**
Yes

**Did it work in any of the earlier releases of .NET Core?**
No response

**Environment Details:**
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 10.0.26100.3624

**More info**
The issue is visible when focus a ToolStripButton with mouse (it draws the keyborad focus as well)
Tried to bypass via custom renderer, not working.

**Repro Steps:**

1. Create a form with a PropertyGrid.
2. Select the main form as "Selected Object"
3. Run the application.
4. Set focus with the mouse on a non selected ToolStripButton.

**Expected behavior:**
Only "Selected" border is visible, on mouse over

![Image](https://github.com/user-attachments/assets/fa7a4130-4554-4442-bca2-6eda0e958b96)

**Actual behavior:**
Selected border and Keyboard focus are visble, on mouse over

![Image](https://github.com/user-attachments/assets/b700ba0f-3f0c-4706-bb5b-b47d04fb9ce1)

**Comment by Tanya-Solyanik** [comment]
@AngeloCresta  - could you please clarify

>Selected border and **Keyboard focus** are visible, on mouse over

Selected border is the dark border around the "alphabetical order" button.
What do you mean by the keyboard focus?

**Comment by AngeloCresta** [comment]
The Button border is not only drawn with the "selected border" color (Blue) when the mouse is over, but it has also the dashed black border:

![Image](https://github.com/user-attachments/assets/753d9e3c-d52f-4bfe-aa48-01589ce95660)

**Comment by AngeloCresta** [comment]
I'll add this to show the inconsistent behaviour between std ToolStrips and the PropertyGrid one:

![Image](https://github.com/user-attachments/assets/60d38b59-d81c-4b72-9208-76fa7223a9ad)

**Comment by merriemcgaw** [comment]
@Olina-Zhang can your team take a look at the keyboard behaviors here? If it has keyboard focus and shows keyboard focus then it's correct. If it's showing keyboard focus but space/enter isn't doing the right thing we should take a look.

@AngeloCresta there will be some inconsistency with the ToolStrip in the PropertyGrid in that the first two buttons are toggles (either you have Category selected or Alphabetical selected) so there needs to be an indicator that a button can be "pushed in" so to speak, as the first button in your image shows.

**Comment by Tanya-Solyanik** [comment]
@AngeloCresta  - this was changed to support accessibility requirements that keyboard focus should be visible.
